components/esp_jpeg/README.md

```diff
@@ -0,0 +1,45 @@
+# JPEG Decoder: TJpgDec - Tiny JPEG Decompressor
+
+TJpgDec is a generic JPEG image decompressor module that highly optimized for small embedded systems. It works with very low memory consumption.
+
+Some microcontrollers have tjpg decoder in ROM, it is used, if there is. [^1] Using ROM code can be disabled in menuconfig.
```

Comment:
Which MCUs exactly have the implementation in ROM?

There should be a paragraph explaining pros and cons of using the ROM implementation (flexibility, configuration, speed?, memory footprint program/data memory)